ui-guide/App/App.purs

```diff
@@ -34,52 +37,68 @@ import Routing (hashes)
 data Query a
   = RouteChange String a
 
-type State =
+type State m =
   { route :: String
+  , stories :: Stories m
+  , partitions :: M.Map Group (Stories m)
```

Comment:
...vs. saving the `partitions` lets us easily render the stories associated with a particular group